core/components/_helpers/globals.js

```diff
@@ -184,6 +184,17 @@ if (includeGlobals) {
       font-style: normal;
     }
 
+    @font-face {
+      font-family: fakt-web;
+      src: url(https://cdn.auth0.com/website/ds/fonts/fakt/FaktPro-SemiBold.woff2)
```

Comment:
Should we use `url("...")` instead of `url(...)`?

Reply:
Makes sense!